### DISCUSSION TITLE
Document of rx js version 6 is not available anymore?

### DISCUSSION THREAD

**Discussion by avri87** [ReactiveX/rxjs#6990] [Report issues other than bug]
Hey,

When I need to see some documentaion about version 6, I get information regarding newer version, for instance, the share operation [link](https://v6.rxjs.dev/api/index/function/share)

**Comment by jakovljevic-mladen** [comment]
Hi, this is now fixed and the V6 docs site is now available. Sorry for the inconvenience.
